stores/sql/sqlite/migrations/main/schema.sql

```diff
@@ -76,6 +76,11 @@ CREATE TABLE `contract_sectors` (`db_sector_id` integer,`db_contract_id` integer
 CREATE INDEX `idx_contract_sectors_db_contract_id` ON `contract_sectors`(`db_contract_id`);
 CREATE INDEX `idx_contract_sectors_db_sector_id` ON `contract_sectors`(`db_sector_id`);
 
+-- dbHost <-> dbSector
+CREATE TABLE `host_sectors` (`db_sector_id` integer,`db_host_id` integer,PRIMARY KEY (`db_sector_id`,`db_host_id`),CONSTRAINT `fk_host_sectors_db_sector` FOREIGN KEY (`db_sector_id`) REFERENCES `sectors`(`id`) ON DELETE CASCADE,CONSTRAINT `fk_contract_sectors_db_host` FOREIGN KEY (`db_host_id`) REFERENCES `hosts`(`id`) ON DELETE CASCADE);
+CREATE INDEX `idx_host_sectors_db_host_id` ON `host_sectors`(`db_host_id`);
+CREATE INDEX `idx_host_sectors_db_sector_id` ON `host_sectors`(`db_sector_id`);
```

Comment:
Probably want to have a unique index over both columns to make sure we don't have duplicate associations